apple/RNGHStylusData.m

```diff
@@ -0,0 +1,26 @@
+//
+//  RNGHStylusData.m
+//  DoubleConversion
+//
+//  Created by Michał Bert on 18/09/2024.
+//
+
+#import "RNGHStylusData.h"
+#import <Foundation/Foundation.h>
+
+@implementation StylusData
+
+- (NSDictionary *)toDictionary
+{
+  NSDictionary *stylusDataObject = @{
+    @"tiltX" : @(_tiltX),
+    @"tiltY" : @(_tiltY),
+    @"altitudeAngle" : @(_altitudeAngle),
+    @"azimuthAngle" : @(_azimuthAngle),
+    @"pressure" : @(_pressure),
+  };
+
+  return stylusDataObject;
+}
```

Comment:
```suggestion
- (NSDictionary *)toDictionary
{
  return @{
    @"tiltX" : @(_tiltX),
    @"tiltY" : @(_tiltY),
    @"altitudeAngle" : @(_altitudeAngle),
    @"azimuthAngle" : @(_azimuthAngle),
    @"pressure" : @(_pressure),
  };
}
```

Reply:
Done in [f482862](https://github.com/software-mansion/react-native-gesture-handler/pull/3113/commits/f48286269904e86688598e135378a5d3850cfdd4)